CODEOWNERS

```diff
@@ -62,26 +62,51 @@ Jenkinsfile @cilium/ci
 Jenkinsfile.nightly @cilium/ci
 monitor/ @cilium/monitor
 monitor/payload @cilium/api
+pkg/annotation @cilium/kubernetes
 pkg/apierror/ @cilium/api
 pkg/apipanic/ @cilium/api
 pkg/apisocket/ @cilium/api
 pkg/bpf/ @cilium/bpf
+pkg/byteorde/ @cilium/bpf
+pkg/client @cilium/api
+pkg/comparator @cilium/bpf
 pkg/completion/ @cilium/proxy
+pkg/components/ @cilium/bpf
+pkg/controller @cilium/agent
+pkg/counter @cilium/bpf
+pkg/datapath @cilium/bpf
+pkg/debugdetection @cilium/agent
+pkg/defaults @cilium/agent
 pkg/endpoint/ @cilium/endpoint
+pkg/endpointmanager/ @cilium/endpoint
 pkg/envoy/ @cilium/proxy
+pkg/fqdn/ @cilium/agent
```

Comment:
policy as well?

Reply:
I was on the fence here. A lot of it is, currently, implementation details in the agent. The policy parts are still in policy (the actual `ToFQDNs` field for both k8s and policy/api). fqdn is more a datapath component, but there isn't a group for that.
I can change it, however, but I'm not sure if the policy group is equipped to review this package right now.